src/OpenTelemetry.Instrumentation.SqlClient/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Add support for Filter expression for .NET Core.
```

Comment:
Should we be saying ".NET Core" or "non .NET Framework Targets"? I think .NET6.0 and the future versions would just be called .NET.

Reply:
https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-and-net-core

![image](https://user-images.githubusercontent.com/66651184/195475827-45614ed6-b020-452c-916e-65b164ff83fa.png)

Reply:
updated